cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/InitCommand.java

```diff
@@ -101,19 +106,21 @@ public void execute() {
         Path projectRoot = ProjectUtils.findProjectRoot(this.userDir);
         if (projectRoot != null) {
             CommandUtil.printError(errStream,
-                    "Directory is already within a Ballerina project :" +
+                    "directory is already within a Ballerina project :" +
                             projectRoot.resolve(ProjectConstants.BALLERINA_TOML).toString(),
                     null,
                     false);
+            CommandUtil.exitError(this.exitWhenFinish);
             return;
         }
 
         // Check if the template exists
         if (!template.equals("") && !CommandUtil.getTemplates().contains(template)) {
             CommandUtil.printError(errStream,
-                    "Template not found, use `bal init --help` to view available templates.",
```

Comment:
Should we have periods or not? @praneesha

Reply:
Better to use the period for complete sentences.